airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/jobs/registry.py

```diff
@@ -19,29 +20,41 @@
 )
 
 
-@op(required_resource_keys={"all_metadata_file_blobs"})
+@op(required_resource_keys={"slack", "all_metadata_file_blobs"})
 def add_new_metadata_partitions_op(context):
     """
     This op is responsible for polling for new metadata files and adding their etag to the dynamic partition.
     """
     all_metadata_file_blobs = context.resources.all_metadata_file_blobs
     partition_name = registry_entry.metadata_partitions_def.name
 
-    new_etags_found = [
-        blob.etag for blob in all_metadata_file_blobs if not context.instance.has_dynamic_partition(partition_name, blob.etag)
-    ]
+    new_files_found = {
+        blob.etag: blob.name for blob in all_metadata_file_blobs if not context.instance.has_dynamic_partition(partition_name, blob.etag)
+    }
 
+    new_etags_found = list(new_files_found.keys())
     context.log.info(f"New etags found: {new_etags_found}")
 
     if not new_etags_found:
         return SkipReason(f"No new metadata files to process in GCS bucket")
 
     # if there are more than the MAX_METADATA_PARTITION_RUN_REQUEST, we need to split them into multiple runs
+    etags_to_process = new_etags_found
     if len(new_etags_found) > MAX_METADATA_PARTITION_RUN_REQUEST:
-        new_etags_found = new_etags_found[:MAX_METADATA_PARTITION_RUN_REQUEST]
-        context.log.info(f"Only processing first {MAX_METADATA_PARTITION_RUN_REQUEST} new blobs: {new_etags_found}")
+        etags_to_process = etags_to_process[:MAX_METADATA_PARTITION_RUN_REQUEST]
+        context.log.info(f"Only processing first {MAX_METADATA_PARTITION_RUN_REQUEST} new blobs: {etags_to_process}")
 
-    context.instance.add_dynamic_partitions(partition_name, new_etags_found)
+    context.instance.add_dynamic_partitions(partition_name, etags_to_process)
+
+    # format new_files_found into a loggable string
+    new_metadata_log_string = "\n".join([f"{new_files_found[etag]} *{etag}* " for etag in etags_to_process])
+
+    PublishConnectorLifecycle.log(
+        context,
+        PublishConnectorLifecycleStage.METADATA_SENSOR,
+        StageStatus.SUCCESS,
```

Comment:
I'd maybe make this an in progress? but not strongly opinionated on that one

Reply:
At this point they are successfully queued! Next step is processing.